packages/interface/src/components/interface-skeleton/index.js

```diff
@@ -19,6 +19,8 @@ import {
 	useViewportMatch,
 	useResizeObserver,
 } from '@wordpress/compose';
+import { useSelect } from '@wordpress/data';
+import { store as blockEditorStore } from '@wordpress/block-editor';
```

Comment:
I don't think we're allowed to use the block editor package inside interface 🤔

Reply:
Probably not -- maybe we shouldn't be storing the pattern category. As in, we don't really care that we have a pattern category selected, we want to tell the interface how to be laid out. So, something like 'wide-sidebar' or 'zoom-out-wide'?

Reply:
good idea!